modules/templates/helper.go

```diff
@@ -66,12 +66,13 @@ func NewFuncMap() template.FuncMap {
 
 		// -----------------------------------------------------------------
 		// time / number / format
-		"FileSize":      base.FileSize,
-		"CountFmt":      base.FormatNumberSI,
-		"TimeSince":     timeutil.TimeSince,
-		"TimeSinceUnix": timeutil.TimeSinceUnix,
-		"DateTime":      timeutil.DateTime,
-		"Sec2Time":      util.SecToTime,
+		"FileSize":       base.FileSize,
+		"CountFmt":       base.FormatNumberSI,
+		"Int64TimeSince": timeutil.Int64TimeSince,
```

Comment:
It doesn't look good. And the  `Delivered` is surprisingly  a bad design since 2015 ......

I would suggest to use `time.Time` to convert `Delivered` in model and then use `TimeSince` to render it directly, then no need to add this template helper function

Reply:
We can add a `type TimeStampNano int64` type , and add an `AsTime` function for it.

```
Delivered TimeStampNano
```

```
{{TimeSince .Delivered.AsTime}}
```

Reply:
Will this cause a db change? 
I didn't convert it into  `time.Time` here because I'm not sure whether this may cause a db change.

Reply:
We have `timeutil.TimeStamp`, maybe we can convert `Delivered`'s type into `timeutil.TimeStamp`?